### DISCUSSION TITLE
wire:model.live is closing mobile keyboard

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [livewire/livewire#8893] [1. Help] [answered]
For some reason when I have a wire:model.live text field, it closes the mobile keyboard with every letter I type

Doesn't seem to do this all the time so I'm not sure what is going on here...

Anyone experience the same?

**Comment by lrljoe** [comment]
Live will cause an update/round trip on every key press.

As far as the mobile browser is concerned, it's a new page, so default focus is resumed.

Debounce it to avoid this. There is rarely a need to not use debounce for a text field

**Reply by EriksonEnterprises** [reply]
Thanks I'll try that. Was trying to do live validation but seems to be acting up too much for now...

**Reply by EriksonEnterprises** [reply, marked as answer]
I figured out the problem...

It was my trixeditor

When the live would fire, Trix would cause the component to lose focus

When I removed that, it worked as expected with live validation
